src/FSharp.Control.TaskSeq/TaskSeq.fs

```diff
@@ -11,10 +11,15 @@ module TaskSeq =
     // Just for convenience
     module Internal = TaskSeqInternal
 
-    let empty<'T> = taskSeq {
-        for c: 'T in [] do
-            yield c
-    }
+    let empty<'T> =
+        { new IAsyncEnumerable<'T> with
+            member _.GetAsyncEnumerator(_) =
+                { new IAsyncEnumerator<'T> with
+                    member _.MoveNextAsync() = ValueTask.False
+                    member _.Current = Unchecked.defaultof<'T>
+                    member _.DisposeAsync() = ValueTask.CompletedTask
+                }
+        }
```

Comment:
Thanks for this, it'll be much faster than relying on the resumable state implementation.